procedures/foreman/release.md.erb

```diff
@@ -44,15 +44,23 @@
   * [ ] Tag the release using [tag.sh](https://gist.github.com/tbrisker/d51cedea51ccbcca00e72c1fc61550fe) `tag.sh <%= full_version %>`
   * [ ] Push: `git push upstream <%= short_version %>-stable --follow-tags`
 * [ ] Run the Jenkins [Tarballs Release](https://ci.theforeman.org/job/tarballs-release/) to create tarballs
-* [ ] [Download](https://github.com/theforeman/theforeman-rel-eng/blob/master/download_tarballs), [inspect](https://github.com/theforeman/theforeman-rel-eng/blob/master/inspect_tarballs), [sign and upload detached signatures](https://github.com/theforeman/theforeman-rel-eng/blob/master/sign_tarballs)
+* [ ] Sign Tarballs
+  * Update release version similar to [here](https://github.com/theforeman/theforeman-rel-eng/commit/2029a9688da00d9c385c3438dd71b594ba5f728e)
```

Comment:
this affects tarballs and rpms, so I'd put it as an own major step before "sign tarballs"